### ISSUE TITLE
"Missing translation" everywhere until reload

### ISSUE THREAD

**Issue by zzottel** [vector-im/riot-web#7847]
Since a few versions of Riot Web, I frequently see "missing translation" for every string in Riot Web when I navigate to the still open Riot tab after opening the browser (FF 63.0.3 Linux). I.e. when I open Riot for the first time in the morning.

I think it doesn't happen every time after opening the browser, but very often. A simple reload (without shift) fixes the problem.

I had seen that on previous Riot Web versions once every time after I had installed a new version, but now this happens without a change of the version.

Seen on self-hosted Riot Web 0.17.8 with FF 63.0.3 Linux.

**Comment by lampholder** [comment]
I remember having seen this, though very infrequently Are you able to reproduce it regularly? What are you doing when you come to use Riot - are you opening a sleeping laptop? Is FF opening Riot as a restored tab, or are you typing the URL into the browser and loading afresh?

**Comment by lampholder** [comment]
P2 for now until we know a bit more about how common this is.

**Comment by zzottel** [comment]
It's usually because Riot Web became so slow before that I had to restart Firefox (will create a Ticket about that later, too). So I close and reopen Firefox. When I then switch to the still-present tab, I see Missing Translation when Riot loads and usually reload the tab at once.

At least today I can say for sure that after closing FF yesterday, sending the computer to sleep, then opening FF and with it, Riot, this morning, there were missing translations, too. (Also without having closed the tab itself, I never do that.)

Just a few seconds ago I closed FF because Riot wasn't reacting anymore. When I reopened it and Riot, I saw Missing Translation, but waited a bit. It seems it didn't even restore CSS correctly, because there were strings shown in a Roman font, and the page became longer and longer.
